tests/unit/AnnualReport.spec.ts

```diff
@@ -42,7 +42,9 @@ describe('Annual Report - Part 1 - UI', () => {
   beforeAll(() => {
     // set configurations
     const configuration = {
-      'VUE_APP_AUTH_WEB_URL': 'https://auth.web.url/'
+      'VUE_APP_AUTH_WEB_URL': 'https://auth.web.url/',
+      'VUE_APP_LEGAL_API_URL': 'https://legal-api.url/',
+      'VUE_APP_LEGAL_API_VERSION_2': 'v2'
     }
     configurationStore.setConfiguration(configuration)
```

Comment:
By the way, for next time (not today), the configuration store is global, so you could set the configurations at the top of this file (eg, after lines 38-39).